### PR TITLE
Added CesiumMapViewClient to control any external link presses

### PR DESCRIPTION
**Pull request by teamagam3**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/44" target="_blank">ReviewNinja

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java [hunk 1]
```diff
@@ -51,16 +51,19 @@
     public CesiumMapView(Context context) {
         super(context);
         init(null, 0);
+        setWebViewClient(new CesiumMapViewClient());
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java]: Duplication of setWebViewClient.
move it to init method !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java [hunk 2]
```diff
@@ -51,16 +51,19 @@
...
     }
 
     public CesiumMapView(Context context, AttributeSet attrs) {
         super(context, attrs);
         init(attrs, 0);
+        setWebViewClient(new CesiumMapViewClient());
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java]: !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java [hunk 3]
```diff
@@ -51,16 +51,19 @@
...
     }
 
     public CesiumMapView(Context context, AttributeSet attrs, int defStyle) {
         super(context, attrs, defStyle);
         init(attrs, defStyle);
+        setWebViewClient(new CesiumMapViewClient());
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java]: !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapViewClient.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package com.teamagam.gimelgimel.app.view.viewer.cesium;
+
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+/**
+ * A {@link WebViewClient} to control our {@link CesiumMapView}
+ */
+public class CesiumMapViewClient extends WebViewClient {
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapViewClient.java:9]: Id suggest that you rename this class to express that it disables links.
Something like DisableLinkWebViewClient or OverrideAnyUrlWebViewClient

Comment (teamagam3) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapViewClient.java:9]: I thought we would use it for other things too.

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java (hunk 2 above)
Comment (teamagam3) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java]: !fixed

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java (hunk 3 above)
Comment (teamagam3) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java]: !fixed

File: app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java (hunk 1 above)
Comment (teamagam3) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java]: !fixed

**Pull request merged**